pipdeptree.py

```diff
@@ -280,8 +280,8 @@ def as_dict(self):
                 'required_version': self.version_spec}
 
 
-def render_tree(tree, list_all=True, show_only=None, frozen=False):
-    """Convert to tree to string representation
+def render_tree(tree, list_all=True, show_only=None, frozen=False, exclude=set()):
```

Comment:
From the python docs - https://docs.python.org/3/reference/compound_stmts.html#function-definitions

Reply:
Hi @naiquevin I changed this is following commit, I'm aware of the minefield when using mutable default args, just didn't think it would matter here as render_tree is only called once, thanks 👍